tests/test_field.py

```diff
@@ -27,6 +28,13 @@ def test_invalid_operations_on_fields():
             op(field1(2), field2(3))
 
 
-def test_random(galois_field):
-    from honeybadgermpc.field import GFElement
-    assert type(galois_field.random()) is GFElement
+def test_sqrt(galois_field):
+    from honeybadgermpc.elliptic_curve import Subgroup
+    field = galois_field
+
+    for i in range(100):
+        num = field(random.randint(0, Subgroup.BLS12_381))
+        while pow(num, (Subgroup.BLS12_381-1)//2) != 1:
```

Comment:
can you add a comment here so as to what this is doing? I am guessing you are trying to find a random number which has a root? What happens when the number does not have a root and you call `sqrt` does it error out? can you please add a test cast to test that?